ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -398,6 +398,70 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-image-registry,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
but yeah @adambkaplan I do not want `always_run` to be `true`

Investigating your link below and the ci-operator doc to make the cfg change to remove this

Reply:
or maybe just change the job after the generator runs ...